### PR TITLE
Fix tcl install in windows

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

**Comment by isuruf** [comment]
cc @fangohr, @takluyver

**Comment by takluyver** [comment]
Aha, so this would give us `tclsh` on Windows as well? Thanks, that would be useful. :-)

**Comment by isuruf** [comment]
Ping @conda-forge/tk

File: recipe/meta.yaml [hunk 1]
```diff
@@ -33,6 +33,7 @@ test:
     - test -f "${PREFIX}/bin/tclsh{{ ver_suffix }}"                                            # [unix]
     - test -f "${PREFIX}/bin/wish{{ ver_suffix }}"                                             # [unix]
     - if not exist %LIBRARY_PREFIX%\\bin\\wish{{ ver_suffix.replace(".", "") }}.exe exit 1     # [win]
+    - if not exist %LIBRARY_PREFIX%\\bin\\tclsh{{ ver_suffix.replace(".", "") }}.exe exit 1     # [win]
```
Comment (jakirkham) [inline recipe/meta.yaml]: nit: Would be nice for the selectors to line up.

**Comment by jakirkham** [comment]
One minor comment. Otherwise LGTM.

**Comment by jakirkham** [comment]
Thoughts @jjhelmus?

**Comment by jjhelmus** [comment]
LGTM, Just waiting for the CI's to pass, then I'll merge this.

**Pull request merged**